tetris.py

```diff
@@ -69,14 +69,15 @@ def run_gui():
     new_game_button3 = tkinter.Button(sidebar, text="start")
     new_game_button3.grid(sticky="n")
 
-    small_board = Board(topbar_canvas, 4, 2, D_GREY, (2, 1), None)
+    small_board = Board(topbar_canvas, 4, 2, D_GREY, (2, 1), None, None)
     main_board = Board(
         game_canvas,
         game_width,
         game_height,
         GREY,
         (int(game_width / 2), -2),
         small_board,
+        topbar_score
```

Comment:
Variable name is not great. How about `topbar_label`?

Reply:
There are three labels, (actually one of them is canvas) that the topbar frame is divided into. topbar_time, topbar_score and topbar_canvas. Don't you think it's better to keep the name?